certexchange/polling/common.go

```diff
@@ -2,6 +2,8 @@ package polling
 
 import (
 	"github.com/benbjohnson/clock"
+	logging "github.com/ipfs/go-log/v2"
 )
 
+var log = logging.Logger("f3-certexchange")
```

Comment:
same system name as main cert exchange, do we want to make it different?

Reply:
Nah, it'll just use the same logger.